src/Microsoft.AspNetCore.Razor.Language/TagHelperBinderSyntaxTreePass.cs

```diff
@@ -345,7 +345,8 @@ public override void VisitTagHelperPrefixDirectiveSpan(TagHelperPrefixDirectiveC
                 TagHelperDirectiveType directiveType)
             {
                 directiveText = directiveText.Trim();
-                if (directiveText.StartsWith("\"", StringComparison.Ordinal) &&
+                if (directiveText.Length >= 2 &&
```

Comment:
Will look through our code base to see if anything jumps out. Merging this for now.